src/zoid/button/util.js

```diff
@@ -7,14 +7,29 @@ export function validateScopes(scopes : $ReadOnlyArray<string>) {
         throw new Error(`Expected props.scopes element to be non-empty`);
     }
 }
+export function validateRedirectUrl(returnurl : string) {
+    if (returnurl === undefined || returnurl.length === 0) {
+        throw new Error(`Expected returnurl to be set`);
+    }
+}
 
 
 export function validateResponseType(responseType : string) {
-    const RESPONSE_TYPES : $ReadOnlyArray<string> = [ 'code', 'code+token', 'code+token+id_token', 'code+id_token' ];
+    const RESPONSE_TYPES : $ReadOnlyArray<string> = [ 'code', 'code+token', 'code+token+id_token', 'code+id_token', 'id_token' ];
     if (!responseType) {
         throw new Error(`Expected props.responseType to be set`);
     }
     if (RESPONSE_TYPES.indexOf(responseType) === -1) {
-        throw new Error(`Invalid value for props.responseType. Allowed values: 'code', 'code+token', 'code+token+id_token', 'code+id_token' `);
+        throw new Error(`Invalid value for props.responseType. Allowed values: 'code', 'code+token', 'code+token+id_token', 'code+id_token','id_token' `);
```

Comment:
Recommend dynamically constructing this error so it just matches the enum of `RESPONSE_TYPES`

Reply:
only code and id_token is been allowed